control-plane/p4RuntimeSerializer.cpp

```diff
@@ -1372,13 +1374,14 @@ getMatchFields(const IR::P4Table* table, ReferenceMap* refMap, TypeMap* typeMap)
             matchType = MatchField::TERNARY;
 
             // If this is a ternary match on the result of an isValid() call, desugar
-            // that to a ternary match against the '$valid$' field of the appropriate
+            // that to a ternary match against the '_valid' field of the appropriate
```

Comment:
We won't need to insert that field any more once we update p4info.proto to treat match fields differently